src/search.rs

```diff
@@ -51,6 +51,8 @@ pub struct SearchResult<T> {
     /// The relevancy score of the match.
     #[serde(rename = "_rankingScore")]
     pub ranking_score: Option<f64>,
+    #[serde(rename = "_rankingScoreDetails")]
+    pub ranking_score_details: Option<Map<String, Value>>,
```

Comment:
Hey, it may be good, but I’m not 100% sure.
@curquiza, what do you think? Should we keep the object as a HashMap of value or should we try to deserialize everything in a nice clean enum?

If we try to deserialize it completely, on one hand, it’s easy to use; on the other hand, in meilisearch, we consider these fields as «non-breaking » so they may change in the future and break the SDK (it already happened once with the cutoff)

Reply:
After checking how we did in the [go SDK](https://github.com/meilisearch/meilisearch-go/pull/528/files), I can confirm that this is the right way to do it!
We didn't specify the exhaustive list of possible variants either in the go sdk